test/tools/ossfuzz/YulToEvmDifferentialFuzzer.cpp

```diff
@@ -52,6 +55,24 @@ using namespace solidity::langutil;
 using namespace solidity::util;
 using namespace std;
 
+namespace
+{
+/// @returns true if there are no recursive functions, false otherwise.
+bool recursiveFunctionExists(Dialect const& _dialect, yul::Object& _object)
```

Comment:
Technically, this would rather be something like:
```suggestion
bool recursiveFunctionHasUnreachableVariables(Dialect const& _dialect, yul::Object& _object)
```
:-).